### ISSUE TITLE
only-natural-blocks no longer works

### ISSUE THREAD

**Issue by Bragos24** [BG-Software-LLC/SuperiorSkyblock2#2113]
### Minecraft's Version

Paper 1.20.1

### Plugin's Version

2024.1-b231

### Describe the bug

Hi, I have a problem with the latest Dev-Build: Only-natural-blocks: true no longer works, players can place and break blocks and the mission progresses

### To Reproduce

Place and break blocks and complete the mission despite only-natural-blocks: true

### Additional Information

[aYbM1iif.txt](https://github.com/user-attachments/files/15876466/aYbM1iif.txt)

**Comment by OmerBenGera** [comment]
Check out latest dev build, let me know if it is fixed

**Comment by Bragos24** [comment]
Hi, sorry for the delay but I had to update to 1.20.4.
Now I am using b260 and only-natural-blocks don't work :(

**Comment by Bragos24** [comment]
Yes, I am using latest dev build.
Here a video:
https://youtu.be/t_NafASm0Ag

**Comment by OmerBenGera** [comment]
I cannot replicate the issue
Are you able to replicate it on a server with only SSB installed?
I am using SuperiorSkyblock v2024.1.267

**Comment by Bragos24** [comment]
Ok, I can try but it will take me a few days, thanks for your support.
The strange thing is that with the 2024.1 stable build everything works perfectly.

**Comment by Bragos24** [comment]
Hi, I tried in a server with only SSB installed.
In the default missions of SSB (for example Miner I: Mine 48 Stone and 16 Coal Ore) only-natural-blocks works.
While if I try with the mission I sent you on Additional Information, only-natural-blocks doesn't work and I don't understand why

**Comment by OmerBenGera** [comment]
Did you try your mission on a server with only SSB installed?
As I said, I cannot replicate it myself. Mind taking another video of the issue with only SSB installed?

**Comment by Bragos24** [comment]
https://youtu.be/K2BZCRMwnH4?si=ZzQGO-cWUasrzS40

(server with only SSB installed)

In the default mission of SSB Miner 1 only-natural-blocks works. In all other missions created by me, it doesn't work
I also tried to create the category "Falegname" in which I copied and pasted the mission Miner_1 but only-natural-blocks did not work there.

I tried copying and pasting the content of my mission in place of the "Miner_1.yml" mission (the default one in SSB, where only-natural-blocks works) and only-natural-blocks worked (not visible in the video).
So I'm thinking that only-natural-blocks only works on the default "Miner" category, or maybe I'm doing something wrong.
Sorry to waste your time and thank you very much for your help.

[falegname_1.txt](https://github.com/user-attachments/files/16430151/falegname_1.txt)
[boscaiolo_1.txt](https://github.com/user-attachments/files/16430176/boscaiolo_1.txt)
[minatore_1.txt](https://github.com/user-attachments/files/16430198/minatore_1.txt)
